lib/hawkular/logger.rb

```diff
@@ -0,0 +1,16 @@
+require 'logger'
+require 'hawkular/env_config'
+
+module Hawkular
+  class Logger
+    def initialize(file = STDOUT)
+      @logger = ::Logger.new(file)
+    end
+
+    def log(message, priority = :info)
+      return unless EnvConfig.log_response?
+
+      @logger.send(priority, message)
```

Comment:
Not really sure if it would be more readable with `@logger.send(priority, message) if EnvConfig.log_response?`. Less code surely, but more readable? WDYT?